cognite/neat/rules/models/_rules/dms_schema.py

```diff
@@ -37,6 +43,21 @@ def from_model_id(cls, client: CogniteClient, data_model_id: dm.DataModelIdentif
         views = dm.ViewList(data_model.views)
         container_ids = views.referenced_containers()
         containers = client.data_modeling.containers.retrieve(list(container_ids))
+        while True:
```

Comment:
Does this require `while` ? and is there any cases where we end up in endless while ?